CRM/Volunteer/Form/VolunteerReport.mgd.php

```diff
@@ -1,7 +1,7 @@
 <?php
 
 /**
- * This file declares managed database records of type "ReportTemplate" and
+ * This file declares managed database records of type "OptionValue" and
```

Comment:
I included this by accident - but it is a change being made across the ReportTemplates as we switch to apiv4